src/EnergyPlus/WaterThermalTanks.cc

```diff
@@ -8577,6 +8568,11 @@ void WaterThermalTankData::CalcDesuperheaterWaterHeater(EnergyPlusData &state, b
             AverageWasteHeat = state.dataHeatBal->HeatReclaimSimple_WAHPCoil(SourceID).AvailCapacity -
                                state.dataHeatBal->HeatReclaimSimple_WAHPCoil(SourceID).HVACDesuperheaterReclaimedHeatTotal;
             DesupHtr.DXSysPLR = state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(SourceID).PartLoadRatio;
+        } else if (DesupHtr.ReclaimHeatingSource == CoilObjEnum::CoilCoolingDX) {
+            AverageWasteHeat =
+                state.dataCoilCooingDX->coilCoolingDXs[DesupHtr.ReclaimHeatingSourceIndexNum].reclaimHeat.AvailCapacity -
+                state.dataCoilCooingDX->coilCoolingDXs[DesupHtr.ReclaimHeatingSourceIndexNum].reclaimHeat.HVACDesuperheaterReclaimedHeatTotal;
```

Comment:
This equation keeps confusing me. The *Total is the sum of all reclaimed heat so far? and AvailableCapacity is the max available? and the AverageWasteHeat is the difference. Just hard to wrap your head around this without seeing where HVACDesuperheaterReclaimedHeatTotal is set, or accumulated, or decremented, etc. in code. I just don't see that. But it's the same as everywhere else so this is great.

    for (auto &num : state.dataHeatBal->HeatReclaimDXCoil(SourceID).HVACDesuperheaterReclaimedHeat)
        state.dataHeatBal->HeatReclaimDXCoil(SourceID).HVACDesuperheaterReclaimedHeatTotal += num;

Reply:
So I left a line out. Zero out the var and then sum it.

    state.dataHeatBal->HeatReclaimDXCoil(SourceID).HVACDesuperheaterReclaimedHeatTotal = 0.0;
    for (auto &num : state.dataHeatBal->HeatReclaimDXCoil(SourceID).HVACDesuperheaterReclaimedHeat)
        state.dataHeatBal->HeatReclaimDXCoil(SourceID).HVACDesuperheaterReclaimedHeatTotal += num;

Reply:
The only thing that worries me here is timing of when this happens versus when HVACDesuperheaterReclaimedHeat is set. Good for now, I guess.